lib/foreman/dynflow/daemon.rb

```diff
@@ -0,0 +1,71 @@
+require 'fileutils'
```

Comment:
I was just trying to replicate the way foreman-tasks work. (https://github.com/theforeman/foreman-tasks/blob/master/lib/foreman_tasks/tasks/dynflow.rake)

@iNecas @ares Does the service in tasks do anything different?